### ISSUE TITLE
No key can be translated

### ISSUE THREAD

**Issue by RoseAsbenhe** [i18next/i18next-xhr-backend#267]
Hello!

I'm trying to load the translations via the XHR backend, but no key can be found in the file (although I can see in the network tab that the json is correctly loaded).

```
  const i18nOptions = {
      // preload: ['en'],
      lng: 'en',
      fallbackLng: 'en',
      ns: 'common',
      debug: true,
      backend: {
        loadPath: '/assets/locales/{{lng}}/{{ns}}.json',
      },
    }

    i18n.use(i18nXHR).init(i18nOptions, () => {
      Backbone.history.start(...)
    })
```

The translation file has the following structure:
```
{
  "translation": {
    "key1": {
       "key2": "value"
   }
}
```

And third, a handlebars helper to get my translations:
```
Handlebars.registerHelper('t', (i18nKey, options = {}) => {
  // props for attributes, e.g. count
  const props = _.extend({}, options.hash, {defaultValue: null})
  const translation = i18n.t(i18nKey, props)

  return new Handlebars.SafeString(translation)
})
```

I installed these versions:

    "i18next": "^10.0.1",
    "i18next-xhr-backend": "^1.4.3",

Any ideas?

**Comment by jamuhl** [comment]
Is there any console output from i18next -> backendConnector related?

**Comment by jamuhl** [comment]
Ah...i think i see....the json file should not include the namespace:

```
{
  "key1": {
       "key2": "value"
   }
}
```

**Comment by jamuhl** [comment]
translation (as long not doing multiLoad) does act as filename to download

**Comment by RoseAsbenhe** [comment]
:+1: Thanks for noticing that. I've already removed the namespace-key before, but it still had not worked. 
Now I've also removed 95% of my translation keys and addes just 4 test-keys and they work. So it's not a problem with the xhr-plugin itself but with my JSON (which has worked as it was boundled within my project and not loaded).

I've got a few shared translations (which I won't to get rid of anyway). I try to find out at which point my translations fail (without any further error console log)
